src/js/program/ad-program-controller.js

```diff
@@ -207,7 +206,6 @@ export default class AdProgramController extends ProgramController {
     set volume(volume) {
         const { mediaController, model, provider } = this;
         model.set('volume', volume);
-        super.volume = volume;
```

Comment:
Instead of removing `super.mute` and `super.volume`, implement them, similar to this change:
https://github.com/jwplayer/jwplayer-commercial/commit/f062a079e6b658e751965f509f5d0a6d6b7f9df2

The issue is that we've optimized our babel and webpack settings to reduce the player size, but it cannot implement subclass `super.setter` calls. So we need to implement the code in those setters inline or change setters `set volume(volume)` to methods `setVolume(volume)`

Reply:
@jnatalzia fyi:
@pajong found another babel "loose" setter override issue. Good news is, these are the last places that match this setter pattern `super\.\w+\s*=`. I'll see if we can add it as a custom ESLint rule.

Reply:
:+1: